src/index.ts

```diff
@@ -58,8 +58,9 @@ const renderWithCarbon = ({
 
 export default function supernova(env: Galaxy) {
   const {
-    flags: { isEnabled: isFlagEnabled },
+    flags: { isEnabled },
   } = env;
+  const isFlagEnabled = isEnabled('PS_15585_SN_TABLE_BASIC_FEATURES');
```

Comment:
This name is too generic. It doesn't say what flag is enabled. Either keep it as before or rename to `areBasicFeaturesEnabled`

Reply:
the flag name is not perfect to begin with, since we are introducing multiple unrelated features. But just `isFlagEnabled` wouldn't say anything to someone seeing the code for the first time